src/Nethermind/Nethermind.Mev/MevPlugin.cs

```diff
@@ -149,7 +152,20 @@ public Task InitRpcModules()
             return Task.CompletedTask;
         }
 
-        public async Task<IBlockProducer> InitBlockProducer(IConsensusPlugin consensusPlugin)
+        public Task<IBlockProducer> InitBlockProducer(IBlockProductionTrigger? blockProductionTrigger = null,
+            ITxSource? additionalTxSource = null)
+        {
+            return Task.Run(() =>
+                    (IBlockProducer)new MevBlockProducer(_childConsensusPlugin!.DefaultBlockProductionTrigger,
+                        _nethermindApi.LogManager,
+                        _blockProducers!.ToArray())
+                );
+        }
+
+        public string SealEngineType { get; } = "MEV";
```

Comment:
Do we need this SealEngineType? I think we need to move only InitBlockProducer interface